r_examples/rstudio-train-tune-deploy-xgboost/abalone-train-tune-deploy-xgboost-model.Rmd

```diff
@@ -0,0 +1,270 @@
+---
+title: "Use RStudio on SageMaker and train, tune and deploy a SageMaker model"
+output: html_notebook
+---
+
+<br>
+
+This is a notebook showcasing how to use RStudio on SageMaker to interact and use other SageMaker services.
+To get started with RStudio, you can bring your existing license for Rstudio Workbench or if you don't have any, you'll need to get one from [RStudio PBC](https://www.rstudio.com/). 
+
+To get started check out [this blog](https://aws.amazon.com/blogs/machine-learning/get-started-with-rstudio-on-amazon-sagemaker/) 
+
+Once you have setup your license and created your SageMaker RStudio domain, you can follow the rest of this notebook.
+
+Interacting with the rest of the SageMaker features and services from R & RStudio is achieved through the SageMaker Python SDK which can be loaded in our environment using [Reticulate](https://rstudio.github.io/reticulate/). Reticulate creates an interface between our R environment and the Python packages we install. 
+<br><br><br>
+
+## Setup
+
+When running this notebook for the first time, you may need to install some necessary packages. The following cell will install all that is needed.
+```{r warning=FALSE, include=FALSE}
+if (!'Metrics' %in% installed.packages()) {install.packages("Metrics")}
+library(reticulate)
+if (!py_module_available("sagemaker")){py_install("sagemaker", pip=TRUE)}
+
+```
+Once this is done, we can load all the libraries we will be needing for this example.
+<br><br>
+Load libraries and initialise some variables that we will need later.
+
+```{r warning=FALSE}
+library(reticulate)
+library(readr)
+library(ggplot2)
+library(dplyr)
+library(stringr)
+library(Metrics)
+
+sagemaker <- import('sagemaker')
+
+session <- sagemaker$Session()
+bucket <- session$default_bucket()
+role_arn <- sagemaker$get_execution_role()
+
+
+```
+
+<br><br>
+
+## Download Data
+
+For this example we will be using the famous abalone dataset as can be found on the [UCI dataset archive](https://archive.ics.uci.edu/ml/datasets/Abalone) where we will create a model to predict the age of an abalone shell based on physical measurements. 
+
+
+
+```{r}
+data_file <- 'http://archive.ics.uci.edu/ml/machine-learning-databases/abalone/abalone.data'
```

Comment:
Can you use the AWS-hosted version which is available in S3?
`s3://sagemaker-sample-files/datasets/tabular/uci_abalone/abalone.csv`

Reply:
thanks - updated this